RosettaCore/src/test/java/com/hubspot/rosetta/RosettaBinderTest.java

```diff
@@ -118,20 +120,17 @@ public void itUsesServiceLoaderToDiscoverModules() {
     bean.setId(50L);
     bean.setName("test");
 
-    assertThat(bind(bean)).isEqualTo(map("id_value", "50", "name_value", "test"));
-    assertThat(bindWithPrefix("prefix", bean)).isEqualTo(map("prefix.id_value", "50", "prefix.name_value", "test"));
-  }
-
-  private static Map<String, Object> map(String... strings) {
```

Comment:
And couldn't really figure out another way to fix `map`, I wasn't sure how to accept the variable arguments, while keeping the type of the argument `Map<String, Object>`. I thought of using `ImmutableMap.<String, Object>builder()` but not only this seems overkill for testing, it also breaks on nulls (which are needed in this case).